openapi-generator.code-workspace

```diff
@@ -45,24 +44,22 @@
 		},
 		"files.trimFinalNewlines": false,
 		"files.trimTrailingWhitespace": true,
-
 		"task.saveBeforeRun": "always",
-
 		"java.autobuild.enabled": false,
 		"java.completion.enabled": true,
 		"java.completion.guessMethodArguments": true,
 		"java.completion.maxResults": 5,
 		"java.format.onType.enabled": true,
-
 		"java.referencesCodeLens.enabled": true,
 		"java.saveActions.organizeImports": true,
 		"java.showBuildStatusOnStart.enabled": true,
-
 		"java.dependency.autoRefresh": true,
 		"java.dependency.refreshDelay": 3000,
 		"java.format.enabled": true,
-
 		"maven.pomfile.autoUpdateEffectivePOM": true,
+		"workspace.isHidden": false,
```

Comment:
why are these change necessary ?

Reply:
I didn't expect vscode to put that change into the workspace file. I'll revert.